src/services/search/dealerSearch.ts

```diff
@@ -0,0 +1,40 @@
+import { postData, Service } from "../../base"
+
+import { Paginated } from "../../types/pagination"
+import { DealerParams } from "../../types/params/dealer"
+
+import { SearchDealer } from "../../types/models/dealer"
+
+const defaultPagination = {
+  page: 0,
+  size: 3,
+}
+
+const searchForDealers = (path, searchQuery: DealerParams) => {
+  const { pagination, sort, query } = searchQuery
+  const { page, size } = pagination
+
+  const sizeOrDefault =
+    parseInt((size || "").toString(), 10) || defaultPagination.size
+  const pageOrDefault =
+    parseInt((page || "").toString(), 10) - 1 || defaultPagination.page
+
+  const body = {
+    pagination: {
+      page: pageOrDefault,
+      size: sizeOrDefault,
+    },
+    sort: [sort],
+    query,
+  }
+
+  return postData(Service.SEARCH, path, body)
+}
+
+export const fetchDealers = async (
```

Comment:
For me it's a bit counter intuitive to call a wrapper for a POST request... don't we normally use post + Function for post requests? Perhaps it's just me... I also find it counter intuitive to use a post call for a data request instead of a query...

Reply:
can you bring that to BE?
https://inventory-search-service.preprod.carforyou.ch/swagger-ui/index.html#/Dealer%20Search/searchUsingPOST

Reply:
this origins in the early days and backend had good reasons that a search must be a POST, which I can't recall anymore